### PR TITLE
Consider transaction currency when calculating total money managed

### PR DESCRIPTION
**Pull request by kewitz**

### CODE REVIEW

File: server/lib/queries.js [hunk 1]
```diff
@@ -1049,6 +1049,24 @@ const getBalances = async (collectiveIds, until = new Date()) =>
...
+const getBalancesInHostCurrency = async (collectiveIds, until = new Date()) =>
+  sequelize.query(
+    `
+      SELECT
+        t."CollectiveId",
+        COALESCE(SUM(ROUND(t."netAmountInCollectiveCurrency" * t."hostCurrencyFxRate")), 0) AS "balance"
+      FROM
+        "Transactions" t
+      WHERE
+        t."CollectiveId" IN (:ids)
```
Comment (znarf) [inline server/lib/queries.js:1061]: While we're at it, we should also join on `HostCollectiveId` to make sure we're only counting Transactions related to the host.

File: server/lib/queries.js [hunk 2]
```diff
@@ -1049,6 +1049,24 @@ const getBalances = async (collectiveIds, until = new Date()) =>
     { type: sequelize.QueryTypes.SELECT, replacements: { ids: collectiveIds, until } },
   );
 
+const getBalancesInHostCurrency = async (collectiveIds, until = new Date()) =>
+  sequelize.query(
+    `
+      SELECT
+        t."CollectiveId",
+        COALESCE(SUM(ROUND(t."netAmountInCollectiveCurrency" * t."hostCurrencyFxRate")), 0) AS "balance"
```
Comment (Betree) [inline server/lib/queries.js:1057]: I just checked and we have some transactions with a null `hostCurrencyFxRate`:

```sql
SELECT * FROM "Transactions" t WHERE "hostCurrencyFxRate" IS NULL AND "deletedAt" IS NULL
```

We probably want to fix these entries and add a non-null constraint on `hostCurrencyFxRate`, but if we're not ready to do it now we could do:
```suggestion
        COALESCE(SUM(ROUND(t."netAmountInCollectiveCurrency" * COALESCE(t."hostCurrencyFxRate", 1))), 0) AS "balance"
```

**Review by Betree — COMMENTED** [review]
Looks good

File: server/lib/queries.js (hunk 2 above)
Comment (znarf) [reply server/lib/queries.js:1057]: Can we run a migration and default hostCurrencyFxRate to 1?

File: server/lib/queries.js (hunk 1 above)
Comment (kewitz) [reply server/lib/queries.js:1061]: Awesome!

File: server/lib/queries.js (hunk 2 above)
Comment (kewitz) [reply server/lib/queries.js:1057]: I fixed the recent existing records manually, they were created in the invoice worker.
Also, I added set default value of hostCurrencyFxRate to 1.

**Pull request merged**